### PR TITLE
Feature/private routing

### PR DESCRIPTION
**Pull request by pseudopilot**

### CODE REVIEW

**Review by Arkant — APPROVED** [review]

File: src/app/core/services/user-profile.service.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default function (app) {
           this.lastName = user.lastName || '';
           this.phone = user.phone || '';
           this.email = user.email;
-          this.role = usersMocksService.userRole;
+          this.role = 'role';
```
Comment (PodstDO) [inline src/app/core/services/user-profile.service.js:15]: I believe You will fix this hard code in next commits and You will use it from constants :)

**Review by PodstDO — APPROVED** [review]

File: src/app/core/services/user-profile.service.js (hunk 1 above)
Comment (pseudopilot) [reply src/app/core/services/user-profile.service.js:15]: I have already created constants roles.USER and roles.ADMIN, but they will come in another PR.

**Review by ElenaBorisenkoO — APPROVED** [review]

File: src/app/core/services/constants.js [hunk 2]
```diff
@@ -13,4 +13,9 @@ export default function (app) {
     app.constant('weatherAPIconstants', {
         'APIkey': 'b0f2a7928e6c418baf01f03c9101c5df'
```
Comment (motifej) [inline src/app/core/services/constants.js:14]: why is it hardcoded?

**Review by motifej — APPROVED** [review]

File: src/app/core/services/constants.js (hunk 2 above)
Comment (pseudopilot) [reply src/app/core/services/constants.js:14]: We'll move it to another place

**Pull request merged**